### ISSUE TITLE
Twitter and LinkedIn are breaked and displayed incorrectly since a few hours

### ISSUE THREAD

**Issue by Ordisoftware** [MrAlex94/Waterfox#1242]
Twitter and LinkedIn are breaked and displayed incorrectly since a few hours in Waterfox 56.2.14 as well as Classic 56.3 but they work with Chrome 77 and Firefox 70. I tried changing the Agent without effect. If I save the html page and open it on Chrome, it works. Windows 7 x64.

![image](https://user-images.githubusercontent.com/22332878/67739279-3bbb5980-fa12-11e9-92ba-7d126180a2f6.png)

![image](https://user-images.githubusercontent.com/22332878/67739283-4118a400-fa12-11e9-89e7-9e385551eca4.png)

**Comment by hawkeye116477** [comment]
Both works correct for me, but I'm on Linux.

**Comment by Ordisoftware** [comment]
I deleted all cookies and disabled all plugins, no change. I deleted the profile folder while keeping plugins and it works but I 'm not able to import json saved bookmarks (I select file and nothing is imported, so I copied the old places.sqlite and it works). It is strange since there is no change in the system. The only thing is that I installed Waterfox Classic 56.3 over 52.2.14 another few hours sooner. I'm not able to say if it is that.
